notebooks/02-ingestion/05-working-with-nested-columns.ipynb

```diff
@@ -326,6 +326,44 @@
     "resp.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### New in Druid 29 - Dynamic `path` in JSON_VALUE\n",
```

Comment:
See other reviews on the approach to versioning

Reply:
applied new versioning strategy